tastyworks/models/watchlists.py

```diff
@@ -23,10 +21,18 @@ class WatchlistGroup(object):
     def __init__(self):
         self.watchlists = {}
 
-    async def load_watchlists(self):
-        request_url = '{}/public_watchlists?include_synthetic=true'.format(
-            BASE_URL
+    async def load_watchlists(self, session):
+        """
+        Gets watchlists
+
+        Args:
+            session (TastyAPISession): The session to use.
+        """
+
+        request_url = '{}/public-watchlists'.format(
+            session.API_url
         )
+        # /watchlists is the end point for user created watchlists
```

Comment:
Would it make sense to move this information into the function docstring above?

Reply:
I think that makes sense, I've moved this locally but waiting on your response to testing before I make any updates to my fork.